src/js/util.js

```diff
@@ -1,6 +1,10 @@
+/* Copyright(c) 2019 Philip Mulcahy. */
 /* Copyright(c) 2018 Philip Mulcahy. */
 /* Copyright(c) 2016 Philip Mulcahy. */
 
+// 2019-10-08 ScottMcNay -- added input-checking to getOrderPaymentURL
+// 2019-10-08 ScottMcNay -- Updated copyright notice.
+
```

Comment:
-> commit message

Reply:
to expand on this - using the "blame" feature in git(hub) it is easy to traverse from a line of code to the commit message to understand why it changed.